frontend/src/App.js

```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react';
+import SignIn from './components/sign_in/sign_in';
+import SignUp from './components/sign_up/sign_up';
```

Comment:
Is the sign_in component here different from the LoginPage component imported below?